WalletWasabi/Gma/QrCodeNet/Encoding/EncodingRegion/VersionInformation.cs

```diff
@@ -16,7 +16,7 @@ internal static class VersionInformation
 		/// Only for version greater than or equal to 7
 		/// </summary>
 		/// <param name="tsMatrix">Matrix</param>
-		/// <param name="version">version number</param>
+		/// <param name="version">Version number</param>
```

Comment:
Remove both params comment here. These comments do not say anything that the code doesn't, so they should be removed.

Reply:
Done.